ngrok/src/tunnel_ext.rs

```diff
@@ -71,7 +78,6 @@ pub trait TunnelExt: Tunnel {
     }
 
     /// Forward incoming tunnel connections to the provided Unix socket path.
-    #[cfg(not(target_os = "windows"))]
     #[instrument(level = "debug", skip_all, fields(path))]
     async fn forward_unix(&mut self, addr: impl AsRef<Path> + Send) -> Result<(), io::Error> {
```

Comment:
Maybe we should rename this to `forward_pipe` since that's a bit more "generic"?

Reply:
I was regretting the previous naming as I did this, I didn't think the windows side would slot in so nicely along side the unix side. Easier to change now rather than later, will update.